proto/sf/substreams/rpc/v2/service.proto

```diff
@@ -130,6 +130,37 @@ message OutputDebugInfo {
 
 message ModulesProgress {
   repeated ModuleProgress modules = 1;
+  repeated Job running_jobs = 2;
+  repeated ModuleStats modules_stats = 3;
+}
+
+message Job {
+    uint64 stage = 1;
+    repeated string modules = 2;
+    uint64 start_block = 3;
+    uint64 stop_block = 4;
+    uint64 processed_blocks = 5;
```

Comment:
I think `processed_block_count` is better since there clearer that is a literal value and not a list of processed block.

Reply:
Applies to other `..._blocks` and `..._bytes` and the like.